tests/e2e/greptimedbcluster_test.go

```diff
@@ -75,6 +75,21 @@ var _ = Describe("Basic test of greptimedb cluster", func() {
 		err = listCluster()
 		Expect(err).NotTo(HaveOccurred(), "failed to list cluster")
 
+		go func() {
+			err = createClusterinBaremetal()
+			Expect(err).NotTo(HaveOccurred(), "failed to create cluster in baremetal")
+		}()
+
+		go func() {
+			time.Sleep(100 * time.Second)
```

Comment:
suggestion: It is not recommended to use the sleep. You can modify it to periodically detect whether the port is connected and set a timeout.